docs/src/topics/authentication.md

```
@@ -158,6 +190,36 @@ Supported algorithms:
+@api.get(
+    "/profile",
+    auth=[JWTAuthentication(cookie="access_token")],
+    guards=[IsAuthenticated()],
+)
+async def profile(request):
+    return {"user_id": request["context"]["user_id"]}
+```
+
+When `cookie=` is set, the token is read from the named cookie only — the
+`Authorization` header is ignored for that backend. The cookie value is the
+raw token, no `Bearer ` prefix needed. Extraction and validation both
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)